software/prng/Makefile

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Not really something for this PR, but it might make sense to put all the common makefile stuff into a global makefile that is included in the individual ones at some point.